packages/salesforcedx-apex-debugger/package.json

```diff
@@ -15,7 +15,7 @@
     "@salesforce/salesforcedx-utils": "57.2.0",
     "async-lock": "1.0.0",
     "faye": "1.1.2",
-    "request-light": "0.2.4",
+    "request-light": "^0.7.0",
```

Comment:
We've wavered on if we should pin to versions or not.  For this update The changelog for request-light appears to follow correct semantic versioning and is really only updated to update dependancies so add the ^ seemed like the right thing to do.